packages/expo-sqlite/src/SQLiteDatabase.ts

```diff
@@ -122,9 +123,15 @@ export class SQLiteDatabase {
    * As long as the transaction is converted into a write transaction,
    * the other async write queries will abort with `database is locked` error.
    *
+   * > **Note:** This function is not supported on web.
```

Comment:
Up to you, but I don't think we need to add this here since each function/class has platforms specified on which they are supported.

For example, in case of this, after adding the platforms below:
![CleanShot 2025-03-31 at 16 51 49](https://github.com/user-attachments/assets/2a7e081e-c710-4456-89ef-256d77fe0dbc)

Reply:
the main reason is that is literally the only api not supported on web. also we recommend to use `withExclusiveTransactionAsync` in most case we then this is not supported on web. so i would like to add a note for it.